submission/distributor.go

```diff
@@ -141,6 +157,9 @@ func (d *Distributor) SubmitToLog(ctx context.Context, logURL string, chain []ct
 	if !ok {
 		return nil, fmt.Errorf("no client registered for Log with URL %q", logURL)
 	}
+	defer func(start time.Time) {
+		rspLatency.Observe(time.Since(start).Seconds(), logURL, string(ctfe.AddPreChainName))
+	}(time.Now())
 	return lc.AddPreChain(ctx, chain)
```

Comment:
Why not populate a responses counter here too?
```golang
rspsCounter = mf.NewCounter("http_rsps", "Number of requests", "logid", "ep", "rc")
...
    sct, err := lc.AddPreChain(ctx, chain)
    // figure out a http.Status code here from err somehow...
    rspsCounter.Inc(logURL, string(ctfe.AddPreChainName), strconv.Itoa(status))
    return sct, err
}

Reply:
That's an interesting one For responses I need to get access to the status. That means introducing metrics into our general client lib as it does not expose http status outside.

Reply:
Ah yes, I left the `figure out a http.Status code` as an exercise for the reader.  Maybe something like:
```golang
status := http.StatusOK
if err != nil {
    status = http.StatusBadRequest  // default to this if status code unavailable
    if err, ok := err.(jsonclient.RspError); ok {
        status = err.StatusCode
    }
}
```

Reply:
Introduced the counter with status-code extraction.